transformations/swap_characters/README.md

```diff
@@ -0,0 +1,37 @@
+# Swap Characters Perturbation
+This perturbation adds noise to all types of text sources (sentence, paragraph, etc.) by randomly swapping any two adjacent characters with a default probability of .05. (e.g. "apple" -> "aplpe", "I like pears" -> "Il ike peasr")
+
+Author name: Taylor Sorensen (email: tsor1313@gmail.com)
+
+## What type of a transformation is this?
+This transformation acts like a perturbation to test robustness. Few letters picked at random are replaced with letters 
+which are at keyboard positions near the source letter. Generated transformations display high similarity to the 
+source sentences i.e. the code outputs highly precise generations. This transformation especially can show robustness to typed data where many common errors are expected.
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
+
+## Previous Work
+This has been a fairly common adversarial augmentation in NLP. It is described and explored in this paper:
+
+```bibtex
```

Comment:
isn't this a survey? Would you be able to add reference(s) to the underlying papers?

Reply:
Yes, I will do that